security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -10,11 +10,40 @@ The {cert-manager-operator} is a cluster-wide service that provides application
 
 These release notes track the development of {cert-manager-operator}.
 
-:FeatureName: The {cert-manager-operator}
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.10.2"]
+== Release notes for {cert-manager-operator} 1.10.2
+
+// TODO: Confirm date closer to GA
+Issued: 2023-03-23
+
+The following advisory is available for the {cert-manager-operator} 1.10.2:
+
+// TODO: Update
+* link:https://access.redhat.com/errata/RHEA-2023:TODO[RHEA-2023:TODO]
+https://errata.devel.redhat.com/advisory/109104
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10[cert-manager project release notes for v1.10].
+
+[id="cert-manager-operator-1.10.2-new-features-and-enhancements"]
+=== New features and enhancements
+
+// * This is the GA release of the {cert-manager-operator}.
+* TODO
+
+[id="cert-manager-operator-1.10.2-bug-fixes"]
+=== Bug fixes
+
+* TODO
```

Comment:
@thejasn @swghosh Do we have a list of bug fixes compared to the old TP version that we can list here?

Reply:
I've cc'ed you on slack.

Reply:
Thanks @thejasn. Can you update each of these with some basic release notes text?

We're usually looking for it in the Cause/Consequence/Fix/Result (CCFR) format and we usually use the "release notes text" field, but I don't have permissions to the CM project to see if that field exists.

Again, doesn't have to be anything spectacular since I will take a pass at updating the actual wording. I just need to know a good summary of what the bug was vs now that it's resolved.